src/doc/src/reference/specifying-dependencies.md

```diff
@@ -183,69 +186,113 @@ you need to specify is the location of the repository with the `git` key:
 regex = { git = "https://github.com/rust-lang/regex.git" }
 ```
 
-Cargo will fetch the `git` repository at this location then look for a
-`Cargo.toml` for the requested crate anywhere inside the `git` repository
-(not necessarily at the root --- for example, specifying a member crate name
-of a workspace and setting `git` to the repository containing the workspace).
+Cargo fetches the `git` repository at that location and traverses the file tree to find
+`Cargo.toml` file for the requested crate anywhere inside the `git` repository. 
+For example, `regex-lite` and `regex-syntax` are members of `rust-lang/regex` repo
+and can be referred to by the repo's root URL (`https://github.com/rust-lang/regex.git`)
+regardless of where in the file tree they reside.
```

Comment:
What was not clear to me is if the crate would have to a member of the root Rust project or can be a crate that lives in a subfolder of some multi-language solution. According to `cargo::ops::cargo_read_manifest::read_package` it can be anywhere. I tried to explain that, but not sure I made it any clearer because the regex example has member crates.

Reply:
It's pretty good now! We can always improve in follow-ups if needed.